src/components/TreeView.vue

```diff
@@ -298,8 +299,8 @@ export default {
       const include = this.suffixPathTokens(this.pathSeparator.concat('.*')).join('|')
       const exclude = this.suffixPathTokens(this.pathSeparator.concat('.*', this.pathSeparator, '.*')).join('|')
       return {
-        include: this.usesWindowsSepator ? this.doubleWindowsSeparator(include) : include,
-        exclude: this.usesWindowsSepator ? this.doubleWindowsSeparator(exclude) : exclude,
+        include: this.usesWindowsSeparator ? this.doubleWindowsSeparator(include) : include,
+        exclude: this.usesWindowsSeparator ? this.doubleWindowsSeparator(exclude) : exclude,
```

Comment:
I would have used two separated computed properties for `include` and `exclude` which themselves would use the `doubleWindowsSeparator` method if needed.

Reply:
This change is cosmetic, I will approve the PR without it.